Frontiers_317_delete_products.py

```diff
@@ -0,0 +1,64 @@
+"""
+Working on a script to clean up demo store. delete all products without images.
+"""
+
+import logging
+import os
+from woocommerce import API
+
+logging.basicConfig(level=logging.INFO)
+
+woo_key = 'WOO_KEY'
+woo_secret = 'WOO_SECRET'
+
+def check_env_variables(variable_name):
+    try:
+        os.environ[variable_name]
+    except KeyError as e:
+        error_message = f"The environment variable must be set: {e}"
+        logging.exception(error_message)
+        raise Exception(error_message)
+
+check_env_variables(woo_key)
+check_env_variables(woo_secret)
+
+wcapi = API(
+    url="http://dev.bootcamp.store.supersqa.com",
```

Comment:
Why dont you make this also an environment variable so that this script can be run for any site?
for example we need to run this for the 'staging' site too. And in jenkins we can use the same script but set different env variables.
makes sense?